simulator/src/main/java/com/hedera/block/simulator/grpc/PublishStreamObserver.java

```diff
@@ -41,6 +41,7 @@ public void onNext(PublishStreamResponse publishStreamResponse) {
     @Override
     public void onError(Throwable throwable) {
         logger.log(Logger.Level.ERROR, "Error: " + throwable.toString());
+        // TODO: Stop the stream, retry, or stop simulation
```

Comment:
TODO

Reply:
We may want to consider a process for these.  Something like this might work:
```
A "TODO" comment is permitted, but it must include a ticket ID.
We prefer the following format.
`@todo(###) description`.
```
This can also be used for `future` feature notes and other items that should refer to issues.

Reply:
✅ Done!

Reply:
@AlfredoG87 noted, I keep this conversation open as a reminder, feel free to resolve it after you open an issue, or if you think that this todo is enough for now. Also @jsync-swirlds has a quite good idea for the convention for the todos, maybe if you open an issue, consider to use his process to tag the todo with the issue number.

Reply:
https://github.com/hashgraph/hedera-block-node/issues/286

Reply:
I've opened a [discussion item](https://github.com/hashgraph/hedera-block-node/discussions/287) to discuss the proposed process and format.

Reply:
@jsync-swirlds +1 from me for the discussion & resolved by @AlfredoG87